app/services/video/provider/vimeo.rb

```diff
@@ -25,6 +25,6 @@ def iframe_url
 
   # L'autoplay est à 1 uniquement parce que l'iframe n'est pas chargée
   def embed_with_defaults
-    "#{iframe_url}?autoplay=1&quality=360p"
+    "#{iframe_url}?autoplay=1&quality=360p&dnt=0"
```

Comment:
Il faut le mettre à `1` ou `true` non?

https://help.vimeo.com/hc/fr/articles/12426260232977-%C3%80-propos-des-param%C3%A8tres-du-lecteur

```suggestion
    "#{iframe_url}?autoplay=1&quality=360p&dnt=1"
```